hw7/functional.py

```diff
@@ -0,0 +1,42 @@
+def sequential_map (*args):
+    *functions , numbers = args
+    for func in functions:
+        numbers = map(func, numbers)
+    return list(numbers)
+
+
+def sequential_map_wc (*args):
+    *functions , numbers = args
+    my_chain = func_chain(*functions)
+    return my_chain(numbers)

```

Comment:
Эта функция уже не работает также как и `sequential_map`, так как тут ты применяешь этот набор функций к контейнеру целиком, а не к отдельным элементам
```suggestion
    return list(map(my_chain, numbers))

```